src/otx/recipe/detection/atss_mobilenetv2.yaml

```diff
@@ -0,0 +1,162 @@
+# @package _global_
+defaults:
+  - override /base: detection
+  - override /data: mmdet
+  - override /model: mmdet
+data:
+  subsets:
+    train:
+      batch_size: 8
+      transforms:
+        - type: LoadImageFromFile
+        - type: LoadAnnotations
+          with_bbox: true
+        - type: MinIoURandomCrop
+          min_ious:
+            - 0.1
+            - 0.3
+            - 0.5
+            - 0.7
+            - 0.9
+          min_crop_size: 0.3
+        - type: Resize
+          scale:
+            - 992
+            - 736
+          keep_ratio: false
+        - type: RandomFlip
+          prob: 0.5
+        - type: PackDetInputs
+          meta_keys:
+            - ori_filename
+            - flip_direction
+            - scale_factor
+            - gt_ann_ids
+            - flip
+            - ignored_labels
+            - ori_shape
+            - filename
+            - img_shape
+            - pad_shape
+    val:
+      batch_size: 1
```

Comment:
Why do we use batch size 1 for validation?

Reply:
In OTX1.5 Some issues happen occasionally when validation batch size is over 1. I'll check it later and update this
Thanks for the comment